content/articles/integrated-dns-providers.md

```diff
@@ -57,3 +59,42 @@ However, record fields not supported by DNSimple or the destination provider may
 1. At DNSimple, go to the <label>Account</label> page, and click the <label>Integrated Providers</label> tab.
 1. Under <label>Linked providers<label>, click the <label>Remove</label> button for the Integrated DNS Provider you want to unlink from your DNSimple account. This will remove the ability to view and manage zones and their DNS records at the linked provider via DNSimple.
 ![Unlink an Integrated DNS Provider](/files/account-integrated-provider-unlink.png)
+
+
+## Adding a Zone to an Integrated DNS Provider {#adding-a-zone-to-an-integrated-dns-provider}
+
+If you have an existing domain name in DNSimple, you can also add it to an Integrated DNS provider from the DNS section of the domain management page. If the Integrated DNS provider has not yet been [linked](/articles/integrated-dns-providers#linking-an-integrated-dns-provider) to your DNSimple account, you will be prompted to do so.
+
+1.  Log in to DNSimple with your user credentials.
+1.  Select the relevant account.
+1.  On the header, click on the <label>Domain Names</label> tab.
+1.  Locate the relevant domain, and click on the name to access the domain page.
+1.  Go to the DNS section by clicking the <label>DNS</label> link on the left.
+1.  Click on the <label>Add</label> link corresponding to the desired Integrated DNS Provider inside the <label>DNS Zone Providers</label> card.
+
+    ![Adding a Zone from DNS Zone Providers](/files/dns-zone-providers-add-zone.png)
+
+1.  If the Integrated DNS Provider is already linked to the DNSimple account, select the desired one (if there is more than one from the same provider type, e.g. Azure) and click <label>Add</label> to add the zone to the provider. For instance, if the provider is Azure, the zone will be added to Azure.
+
+    ![Adding a Zone to an already linked DNS Provider](/files/dns-zone-providers-add-to-linked-provider.png)
+1.  If the Integrated DNS Provider is not already linked to the DNSimple account, you will have to input the necessary credentials to link it to the account before the zone can be added to the provider. The Integrated DNS Provider will be linked to the DNSimple account and the zone will then be added to the provider. Note: For [CoreDNS](/articles/integrated-dns-provider-coredns), the provider needs to already be [linked](/articles/integrated-dns-providers#linking-an-integrated-dns-provider) to your DNSimple account.
```

Comment:
> Note: For [CoreDNS](/articles/integrated-dns-provider-coredns), the provider needs to already be [linked](/articles/integrated-dns-providers#linking-an-integrated-dns-provider) to your DNSimple account.

Unless I misunderstood, it should still work even if CoreDNS is not already linked to the account. The user just needs to link using the same flow as the other providers.

Unlinked CoreDNS:
<img width="489" alt="image" src="https://github.com/dnsimple/dnsimple-support/assets/47726656/5e9207b7-2cc0-49b0-9579-5b6851e0ba77">

vs

Linked CoreDNS:
<img width="488" alt="image" src="https://github.com/dnsimple/dnsimple-support/assets/47726656/9427681c-bae7-42ae-baa1-d8f77baf2c76">

Reply:
@kojdm Thanks for the clarification! Addressed in https://github.com/dnsimple/dnsimple-support/pull/1148/commits/644fea7a5b8d300afd3fe2e5d127948f7de96f0d